board/Kconfig

```diff
@@ -1,172 +1,185 @@
 menu "Device Support"
-	config APPLE_N61AP
```

Comment:
1st. Use a proper commit message and commit title, following the lkml style
2nd. You shouldn't be mass changing indentation without any reason
3rd. Add last entry to the end of the list

Best regards, Ivo.